finalPrep/modZeroHero.js

```diff
@@ -55,30 +81,64 @@ var excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.
 //   - citiesDestroyed (array)
 //   - luckyNumbers (array)
 //   - address (object with following key/values: number , street , state, zip)
-
+var scaryMonster = {
+  name: "Grumpus",
+  smell: "sour milk",
+  weightInPounds: 2,
+  cittiesDestroyed: ["Phoenix", "Dallas", "Miami",],
+  luckyNumbers: [23, 9, 544, 48],
+  address: address = {
+    number: 544,
+    street: "Grumpy Boulevard",
+    state: "Grumplevania",
+    zip: 91982,
+  }
```

Comment:
to assign the key `address` to a value which is an object data type you just need:

```
 address: {
    number: 544,
    street: "Grumpy Boulevard",
    state: "Grumplevania",
    zip: 91982,
  }
```